### PR TITLE
docs(user): Add quick comparision of Capella co-working methods

### PR DESCRIPTION
**Pull request by MoritzWeber0**
The co-working methods TeamForCapella and Git are compared in a table.

### CODE REVIEW

File: docs/user/docs/settings/tools/capella/t4c-git-compare.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  </tr>
+  <tr>
+    <td><b>Automation</b></td>
+    <td>Possible via CI/CD in the Git repository.</td>
+    <td>Possible via TeamForCapella &rarr; Git synchronisation (runs on a nightly basis).</td>
+  </tr>
+  <tr>
+    <td><b>Change control</b></td>
+    <td>Reviews are possible via merge requests.</td>
+    <td>Change control is nearly impossible, no barriers.</td>
```
Comment (Paula-Kli) [inline docs/user/docs/settings/tools/capella/t4c-git-compare.md:60]: What is meant with `no barriers` - does something like `no reviews and feedback barriers built in`?

File: docs/user/docs/settings/tools/capella/t4c-git-compare.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  </tr>
+  <tr>
+    <td><b>Release tagging</b></td>
+    <td>Directly possible via tags in Git.</td>
+    <td>Possible via TeamForCapella &rarr; Git synchronisation (Releases are stored as tags in the Git repository).</td>
```
Comment (Paula-Kli) [inline docs/user/docs/settings/tools/capella/t4c-git-compare.md:65]: what is meant by "the" Git repository - are we not talking about TeamForCapella here?

**Review by Paula-Kli — COMMENTED** [review]
This table is a good idea to get a good overview over the different working modes and what consequences it has choosing one or the other in short words 👍

File: docs/user/docs/settings/tools/capella/t4c-git-compare.md (hunk 1 above)
Comment (MoritzWeber0) [reply docs/user/docs/settings/tools/capella/t4c-git-compare.md:60]: There are `no barriers` in TeamForCapella. Everybody with access can push.

Comment (Paula-Kli) [reply docs/user/docs/settings/tools/capella/t4c-git-compare.md:60]: ah I see - so I would probably rather write something like `no access restrictions which can lead to unreviewed and unaligned overwrites` ? Is that what you want to say?

File: docs/user/docs/settings/tools/capella/t4c-git-compare.md (hunk 2 above)
Comment (MoritzWeber0) [reply docs/user/docs/settings/tools/capella/t4c-git-compare.md:65]: This is possible via the `TeamForCapella` > `Git` synchronization. The content is synchronized on a nightly basis from T4C to Git, so tags can still be created in the Git repository.

File: docs/user/docs/settings/tools/capella/t4c-git-compare.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  </tr>
+  <tr markdown="1">
+  <!-- This formatting is required, otherwise mkdocs breaks -->
+  <td><b>License</b></td>
+  <td markdown="1">No license needed[^1]</td>
+  <td markdown="1">TeamForCapella license required[^2]</td>
+  </tr>
+  <tr>
+    <td><b>Complexity</b></td>
+    <td>High complexity, Git knowledge is required.</td>
```
Comment (amolenaar) [inline docs/user/docs/settings/tools/capella/t4c-git-compare.md:39]: The complexity is not high, as long as you can stay with commit/push/pull only.

File: docs/user/docs/settings/tools/capella/t4c-git-compare.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    <td>Changes are backed up on a nightly basis.</td>
+  </tr>
+  <tr>
+    <td><b>Automation</b></td>
+    <td>Possible via CI/CD in the Git repository.</td>
+    <td>Possible via TeamForCapella &rarr; Git synchronisation (runs on a nightly basis).</td>
+  </tr>
+  <tr>
+    <td><b>Change control</b></td>
+    <td>Reviews are possible via merge requests.</td>
```
Comment (amolenaar) [inline docs/user/docs/settings/tools/capella/t4c-git-compare.md:59]: Conducting a review on a merge request of a capella model sounds challenging to me. Or is there an easy way to see what has changed? I assume the Eclipse diff/merge tool will not be very helpful here.

**Review by amolenaar — APPROVED** [review]

File: docs/user/docs/settings/tools/capella/t4c-git-compare.md (hunk 3 above)
Comment (MoritzWeber0) [reply docs/user/docs/settings/tools/capella/t4c-git-compare.md:39]: > as long as you can stay with commit/push/pull only

Our projects have shown that this is more of a wishful thinking. Sooner or later at least a merge will be necessary.
I tried to clarify it a bit more - "High complexity" was removed, it just says "Git knowledge is required". A footnote has been added with the following content:

```
For the basic workflow, a simple Git knowledge that includes commit, pull
    and push is sufficient. However, it is necessary to have at least one Git
    expert in the project to deal with potential merge conflicts and unexpected
    situations.
```

File: docs/user/docs/settings/tools/capella/t4c-git-compare.md (hunk 4 above)
Comment (MoritzWeber0) [reply docs/user/docs/settings/tools/capella/t4c-git-compare.md:59]: The Eclipse diff/merge tool has the capability to compare two different revisions. However, the list of changes can be hard to review, as every single moved diagram elements is listed as change.

Comment (MoritzWeber0) [reply docs/user/docs/settings/tools/capella/t4c-git-compare.md:59]: Added a bit more details about the complexity of the EMF diff/merge.

**Pull request merged**